imports/plugins/included/jobcontrol/server/jobs/cart.js

```diff
@@ -0,0 +1,91 @@
+import later from "later";
+import moment from "moment";
+import { Accounts, Cart, Jobs } from "/lib/collections";
+import { Hooks, Logger, Reaction } from "/server/api";
+import { ServerSessions } from "/server/publications/collections/sessions";
+
+
+Hooks.Events.add("afterCoreInit", () => {
+  Logger.debug("Adding Job removeStaleCart and Accounts to jobControl");
+  const settings = Reaction.getShopSettings();
+  if (settings.cart) {
+    new Job(Jobs, "cart/removeFromCart", {})
+      .priority("normal")
+      .retry({
+        retries: 5,
+        wait: 60000,
+        backoff: "exponential" // delay by twice as long for each subsequent retry
+      })
+      .repeat({
+        schedule: later.parse.text("every day")
+      })
+      .save({
+        cancelRepeats: true
+      });
+  } else {
+    Logger.warn("No cart cleanup schedule");
+  }
+});
+
+/**
+ * {Function} that fetches stale carts
+ * @param {Object} olderThan older than date
+ * @return {Object} stale carts
+ */
+const staleCarts = (olderThan) => {
```

Comment:
this function should be called `getStaleCarts`. Function names should always contain verbs